tests/test_builder.py

```diff
@@ -0,0 +1,335 @@
+from mandible.metadata_mapper.builder import (
+    add,
+    build,
+    floordiv,
+    mapped,
+    mul,
+    reformatted,
+    sub,
+    truediv,
+)
+
+
+def test_build_noop():
+    template = {
+        "something": ["simple"],
+        "with": {
+            "nested": [
+                {"data": ["structures"]}
+            ]
+        }
```

Comment:
It is a good habit to review our coding standards on occasion. Our coding standards state that the last item in a dict should have a trailing comma.

Reply:
I'll go through the files and check them. Much of this was actually moved code that I didn't modify or copied from code that was written before anyone was paying attention to trailing commas at all.

BTW I re-read our coding standards as recently as last week. Did you?

Reply:
Yep. Weirdly enough, I read it last week, too. I also read it today.